packs/data/classfeatures.db/bloodline-wyrmblessed.json

```
@@ -54,6 +54,209 @@
                 "path": "data.proficiencies.aliases.sorcerer",
                 "priority": 39,
                 "value": "divine"
+            },
+            {
+                "adjustName": true,
+                "choices": [
+                    {
+                        "label": "Black Dragon",
```

```suggestion
                        "label": "PF2E.SpecificRule.DragonDisciple.Black",
```
We should localize all these, Dragon disciple already has a good chunk of them in re-en.json so let's just expand that list to include all of them.